### PR TITLE
Rewriting the Silo code in fsharp

### PR DESCRIPTION
**Pull request by AmedeoV**
image

### CODE REVIEW

File: 5a/SiloHost/src/Program.fs [hunk 1]
```diff
@@ -116,8 +124,35 @@ let main args =
     printfn $"Gateway Port: {gatewayPort}"
     printfn $"Primary Silo Port: {primarySiloPort}"
     printfn $"Dashboard Port: {dashboardPort}"
-
-    //TODO Add Orleans host
-
+    
+    let configureLogging (builder : ILoggingBuilder) =
```
Comment (AmedeoV) [inline 5a/SiloHost/src/Program.fs:128]: This works for now, but I guess we can make it prettier...

File: 5a/SiloHost/SiloHost.fsproj [hunk 2]
```diff
@@ -9,13 +9,10 @@
 
     <ItemGroup>
         <PackageReference Include="Microsoft.Extensions.Hosting" Version="6.0.0" />
-<!--        <PackageReference Include="Microsoft.Orleans.CodeGenerator.MSBuild" Version="3.6.0">-->
```
Comment (AmedeoV) [inline 5a/SiloHost/SiloHost.fsproj:12]: removed the commented out libraries, we can add them when/if we need them

**Review by askpt — APPROVED** [review]
LGTM!

File: 5a/SiloHost/src/Program.fs (hunk 1 above)
Comment (PiotrJustyna) [reply 5a/SiloHost/src/Program.fs:128]: Ah it's fine 👍

**Review by PiotrJustyna — APPROVED** [review]
Nice work, LGTM! We can always polish it up later. Glad to see this working.

**Pull request merged**